### DISCUSSION TITLE
Gone are the days

### DISCUSSION THREAD

**Discussion by JamesStallings** [intelligencedev/eternal#74] [General]
Gone are the days when a 'nerd' can claim 'I just do (insert nerd skill), I can't do art'

I never subscribed to that view anyway, so I feel somewhat vindicated as a writer, coder, artist, and content maker. The poetry was epic, BTW.

**Comment by Art9681** [comment]
> God creates dinosaurs. God destroys dinosaurs. God creates man. Man destroys God. Nerd creates AI. AI augments nerd. Nerd inherits the earth.
